src/main/java/com/worldline/codetest/rest/ProfileApi.java

```diff
@@ -0,0 +1,52 @@
+package com.worldline.codetest.rest;
```

Comment:
Nice Refactoring, Moving the Swagger Specification to an interface

Reply:
JAX-RS annotations and swagger makes classes hard to read.

Im fan of interfaces, especially default methods in Interfaces ( Java8 ), wish more start using it.